zebra-chain/src/sapling/keys.rs

```diff
@@ -958,9 +1001,16 @@ impl FromStr for FullViewingKey {
     }
 }
 
-/// An ephemeral public key for Sapling key agreement.
+/// An [ephemeral public key][1] for Sapling key agreement.
 ///
-/// https://zips.z.cash/protocol/protocol.pdf#concretesaplingkeyagreement
+/// Public keys containing points of small order are not allowed.
+///
+/// It is denoted by `epk` in the specification. (This type does _not_
+/// represent [KA^{Sapling}.Public][2], which allows any points, including
+/// of small order).
+///
+/// [1]: https://zips.z.cash/protocol/protocol.pdf#outputdesc
+/// [2]: https://zips.z.cash/protocol/protocol.pdf#concretesaplingkeyagreement
```

Comment:
It's annoying that this distinction has to be made, I wonder if we can/should update the spec

Reply:
I'm not sure how :thinking: